packages/lrzip/configure.ac.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/configure.ac b/configure.ac
+index 4b12022..4714ef4 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -108,7 +108,7 @@ AC_C_INLINE
+ AC_C_BIGENDIAN
+ 
+ AC_CHECK_LIB(pthread, pthread_create, ,
+-        AC_MSG_ERROR([Could not find pthread library - please install libpthread]))
++        AC_MSG_NOTICE([Could not find pthread library - please install libpthread]))
```

Comment:
Bionic libc does not have `libpthread.*` files. You can omit the entire `AC_CHECK_LIB` code here.